src/lib/storage/resolve_encoded_column_type.hpp

```diff
@@ -41,20 +43,20 @@ void resolve_encoded_column_type(const BaseEncodedColumn& column, const Functor&
   // Iterate over all pairs in the map
   hana::fold(encoded_column_for_type, false, [&](auto match_found, auto encoded_column_pair) {
     const auto encoding_type_c = hana::first(encoded_column_pair);
-    const auto column_template_t = hana::second(encoded_column_pair);
+    const auto column_template_c = hana::second(encoded_column_pair);
 
     constexpr auto encoding_type = hana::value(encoding_type_c);
 
     // If the column’s encoding type matches that of the pair, we have found the column’s type
     if (!match_found && (encoding_type == column.encoding_type())) {
       // Check if ColumnDataType is supported by encoding
-      const auto data_type_supported = encoding_supports_data_type(encoding_type_c, hana::type_c<ColumnDataType>);
+      const auto data_type_supported = encoding_supports(encoding_type_c, hana::type_c<ColumnDataType>);
```

Comment:
I don‘t like this change. Supports... what?

Reply:
I thought it is clear because of the parameter.

Reply:
Hmm, as soon as `hana::type_c` comes into play, I think we should rather be more verbose.